PurchasesTests/Identity/IdentityManagerTests.swift

```diff
@@ -86,21 +87,22 @@ class IdentityManagerTests: XCTestCase {
         expect(self.mockBackend.invokedCreateAlias).toEventually(beTrue())
     }
 
-    func testCreateAliasNoOpsIfNilAppUserID() {
+    #if arch(x86_64)
+    //  Also, Nimble's throwAssertion() matcher doesn't work for ARM64.
+    //  See https://github.com/Quick/Nimble/blob/main/Sources/Nimble/Matchers/ThrowAssertion.swift#L125
+    func testCreateAliasFatalErrorsIfCurrentAppUserIDIsNil() {
         self.mockBackend.invokedCreateAlias = false
         self.mockDeviceCache.stubbedAppUserID = nil
-        self.identityManager.createAlias(appUserID: "cesar"){ (error: Error?) in
-        }
-
-        expect(self.mockBackend.invokedCreateAlias).toEventually(beFalse())
+        expect(self.identityManager.createAlias(appUserID: "cesar"){ _ in }).to(throwAssertion())
     }
+    #endif
 
     func testCreateAliasCallsCompletionWithErrorIfNilAppUserID() {
         self.mockBackend.invokedCreateAlias = false
-        self.mockDeviceCache.stubbedAppUserID = nil
+        self.mockDeviceCache.stubbedAppUserID = "cesar"
         var completionCalled = false
         var receivedNSError: NSError?
-        self.identityManager.createAlias(appUserID: "cesar"){ (error: Error?) in
+        self.identityManager.createAlias(appUserID: ""){ (error: Error?) in
```

Comment:
Switch it up, since we can't have a nil currentAppUserID, let's pretend we have `cesar` and we attempt to alias an empty String.